tests/braid_tests.rs

```diff
@@ -4,8 +4,8 @@ use std::{
     path::Path,
 };
 
-use braidpool::braid::io_json::{check_cohort, load_braid, save_braid};
```

Comment:
This test file is not being run by `cargo test`. I just pushed a patch to get it to run.